app/models/customer.py

```diff
@@ -0,0 +1,46 @@
+from flask import current_app
+from sqlalchemy.sql.expression import true 
+from app import db
+from datetime import datetime 
+class Customer(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String, nullable=False)
+    postal_code = db.Column(db.Integer, nullable=False)  
+    phone = db.Column(db.String, nullable=True) 
+    # date/time when video was released 
+    registered_at = db.Column(db.DateTime, \
+        default=datetime.now(),
+        nullable=False) 
+    videos_checked_out_count = db.Column(db.Integer, default=0) 
```

Comment:
Instead of `videos_checked_out_count` being a column we can use the rentals to count that.  

To do that it's helpful to add a relationship:
```rentals = db.relationship('Rental', back_populates='customer', lazy=True)```

For `to_dict`, once you have the relationship the `videos_checked_out_count` is the length of the `rentals`:
```videos_checked_out_count": len(self.rentals)```

This code wouldn't work exactly as written with your implementation since a rental is created both at check-in and check-out. The code written here assumes a rental is deleted when it was checked it. We might also consider changing the status of the rental to "checked-in" and then writing logic just to find rentals that are checked out.

This method of dynamically computing `videos_checkout_out_count` rather than storing it in an instance variable makes it so there isn’t information stored in multiple places that could potentially conflict.